docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -285,4 +285,28 @@ kotlin {
         }
     }
 }
-```
\ No newline at end of file
+```
+
+### Sharing Kotlin cinterop between dependent Pods
+
+If you're adding multiple dependencies on Pods using the `pod()` function, you might encounter issues in
+case there are dependencies between APIs of your Pods.
+
+To make the code compile in this case, use the `useInteropBindingFrom()` function.
+It utilizes the cinterop binding generated for another Pod when building a binding for the new Pod.
+
+The dependent Pod should be declared before setting up the dependency:
```

Comment:
It's important to declare the dependent Pod before setting up the dependency:

Does this work?

Reply:
You should declare the dependent Pod before setting up the dependency: